docs/integrations/azure/org.md

```diff
@@ -258,21 +258,96 @@ The `myUserTransformer`, `myGroupTransformer`, `myOrganizationTransformer`, and
 
 The following provides an example of each kind of transformer. We recommend creating a `transformers.ts` file in your `packages/backend/src` folder for these.
 
+First, lets set up the basic structure of the file, with functions for each kind of transformer that simply passes through the default transformer unchanged.
+
 ```ts title="packages/backend/src/transformers.ts"
```

Comment:
```suggestion
```ts title="packages/backend/src/extensions/transformers.ts"
```

We followed this pattern for the Permission docs and tutorial, might make sense to keep this consistent.

Reply:
We also do it on the Demo site: https://github.com/backstage/demo/blob/master/packages/backend/src/extensions/docsBuildStrategy.ts